src/Widgets/WallpaperContainer.vala

```diff
@@ -85,25 +68,24 @@ public class WallpaperContainer : Gtk.FlowBoxChild {
     }
 
     construct {
-        scale = get_style_context ().get_scale ();
+        var style_context = get_style_context ();
+        style_context.add_class ("wallpaper-container");
+
+        scale = style_context.get_scale ();
 
         height_request = THUMB_HEIGHT + 18;
         width_request = THUMB_WIDTH + 18;
 
         var provider = new Gtk.CssProvider ();
-        try {
-            provider.load_from_data (CARD_STYLE_CSS, CARD_STYLE_CSS.length);
-            Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
-        } catch (Error e) {
-            critical (e.message);
-        }
+        provider.load_from_resource ("/io/elementary/switchboard/plug/pantheon-shell/plug.css");
+        Gtk.StyleContext.add_provider_for_screen (Gdk.Screen.get_default (), provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
```

Comment:
Since the new gtk, `add_provider` for a widget seems to not be applying the style for it's childs, only to the widget itself. At least that's what's happened with Spice-Up and had to jump though some hoops to get all widgets styled correctly